### ISSUE TITLE
Changing text/underline color

### ISSUE THREAD

**Issue by zmajeric** [dkharrat/NexusDialog#45]
I've look through entire code, couldn't find a simple way to change color to white (from black). Is kind of stupid to create custom element just to change color, or is it really the only way?

**Comment by dkharrat** [comment]
There are 2 possible ways:

1. Call `getView()` on the element controller to get access to the underlying `View` of the element.
2. You can override the .xml layout for a specific element by adding your own .xml layout with the same name. For example, if you want to override the style for the text elements, add a `form_labeled_element.xml` layout in your own project, which will override the one in NexusDialog.

**Comment by zmajeric** [comment]
Wow didn't expect such thorough response,  appreciate it. Actually I tried with opt. 1 but in the stage of creating form, getView (or eg. getEditText) returned null pointer. Did not know opt. 2 is possible, but I solved it by assigning specific theme to this exact activity and change xml attributes to match my design as much as possible. While I'm at it is there a way to remove top section line.? Otherwise just call it closed.
